Adding some additional clarification for mentees. (#672)
jaegertracing/documentation: content/mentorship-for-mentees.md

In order to understand the project better and come up with reasonable solutions, it's always helpful to become familiar with Jaeger and its code base. We strongly recommend:

> * Going through some Jaeger tutorials, such as [this one][hotrod].

Reviewer: ```suggestion
* Going through some Jaeger tutorials, such as [HotROD demo][hotrod] or [YouTube video][hotrod_video]
```

> * Review the hotrod video demo and tutorial such as [this video on youtube][hotrod_video].

Reviewer: ```suggestion
* Review the HotROD video demo and tutorial such as [this video on youtube][hotrod_video].
```
HotROD ™️

> * Run the [hotrod demo yourself][hotrod_repo] as the blogs and video is outdated, it's good to get hands on.

Reviewer: ```suggestion
* Run the [HotROD demo yourself][hotrod_repo] as the blogs and video is outdated, it's good to get hands on.
```
HotROD ™️

> * Review the Jaeger architecture from the [documentation](https://www.jaegertracing.io/docs/architecture/) and understand the microservices architecture.

Reviewer: > understand the microservices architecture.

what do you mean by that?

> * Review the Jaeger architecture from the [documentation](https://www.jaegertracing.io/docs/architecture/) and understand the microservices architecture.

Reviewer: ```suggestion
* Review the [Jaeger architecture documentation](https://www.jaegertracing.io/docs/architecture/) and understand the microservices architecture.
```

> understand the microservices architecture.

what do you mean by that?

Reviewer: > https://www.jaegertracing.io/docs/architecture/

better to use relative path

Author: Understanding of the components in the architecture is another way to phrase it, if you prefer.

I will edit to use relative on the push.
